src/storage/src/client.rs

```diff
@@ -132,17 +134,19 @@ impl Storage {
     /// # use google_cloud_storage::client::Storage;
     /// async fn example(client: &Storage) -> gax::Result<()> {
     ///     let contents = client
-    ///         .read_object()
-    ///         .set_bucket("projects/_/buckets/my-bucket")
-    ///         .set_object("my-object")
+    ///         .read_object("projects/_/buckets/my-bucket", "my-object")
     ///         .send()
     ///         .await?;
     ///     println!("object contents={contents:?}");
     ///     Ok(())
     /// }
     /// ```
-    pub fn read_object(&self) -> ReadObject {
-        ReadObject::new(self.inner.clone())
+    pub fn read_object<B, O>(&self, bucket: B, object: O) -> ReadObject
```

Comment:
non-blocking: Might be nice to document the required parameters like we used to do in storage-control:

https://github.com/googleapis/google-cloud-rust/blob/dbe7a0e152fe0cbc2b4f0b3b9bed14f375670a42/src/storage-control/src/client.rs#L197-L200

Reply:
Done!